samples/excel/89-preview-apis/style.yaml

```diff
@@ -0,0 +1,241 @@
+order: 10
+id: excel-style
+name: Style
+description: 'Add, apply, get and delete styles.'
+author: siewmoi
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-new-style").click(() => tryCatch(addNewStyle));
+        $("#apply-new-style").click(() => tryCatch(applyNewStyle));
+        $("#apply-built-in-style").click(() => tryCatch(applyBuiltInStyle));
+        $("#get-style-font").click(() => tryCatch(getStyleFontProperties));
+        $("#get-style-alignment").click(() => tryCatch(getStyleAlignmentProperties));
+        $("#delete-new-style").click(() => tryCatch(deleteNewStyle));
+
+        async function addNewStyle() {
+            await Excel.run(async (context) => {
+                let styles = context.workbook.styles;
+
+                // Add a new style to the style collection.
+                // Styles is in the Home tab ribbon.
+                styles.add("Diagonal Orientation Style");
+                styles.getItem("Diagonal Orientation Style").orientation = 38;
+                styles.getItem("Diagonal Orientation Style").addIndent = true;
+               
+                let newStyle = styles.getItem("Diagonal Orientation Style");
+                context.load(newStyle);
+
+                newStyle.includeProtection = true;
+                newStyle.shrinkToFit = true;
+                newStyle.locked = false;
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("Successfully added a new style with diagonal orientation to the Home tab ribbon.");
+            });
+        }
+
+        async function applyNewStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply new style.
+                worksheet.getRange("A1:E1").style = ("Diagonal Orientation Style");
+                worksheet.getRange("A1:E1").format.verticalAlignment = "Justify";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function applyBuiltInStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply built-in style. 
+                // Styles are in the Home tab ribbon.
+                worksheet.getRange("A1:E1").style = Excel.BuiltInStyle.neutral;
```

Comment:
The code gets a range object for the exact same range 3 times. You can get it once and assign to a variable. Then set the style and verticalAlignment properties using the variable.